### PR TITLE
add realwingame gui

### PR DESCRIPTION
**Pull request by lordIcocain**
upd video
https://youtu.be/DedMOKb0qBg

### CODE REVIEW

**Comment by vovaksenov** [comment]
Not sure where you got this list and what it is used for, but it might contain inaccuracies. Prometheus0000 im pretty sure is a dev/staff, not a contributor for example.

File: src/main/resources/assets/dreamcraft/RealWinGameData.txt [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+Cleptomania:contributor
+RaitGamerGR:contributor
+ko5h:contributor
+DualBallers:contributor
+YoungGT:contributor
+Yesterday17:contributor
+
+Pandoro:stargate
+JodieRuth:stargate
+PropainChan
```
Comment (LewisSaber) [inline src/main/resources/assets/dreamcraft/RealWinGameData.txt:160]: typo?

**Comment by lordIcocain** [comment]
> Not sure where you got this list, but it might contain inaccuracies. Prometheus0000 im pretty sure is a dev/staff, not a contributor for example.

my dev source is discord rolemembers  role GTNH Devs and GT:NH Dev on Hiatu, so yeah mb have error jsut because i dont know all dev ppl

**Comment by vovaksenov** [comment]
I think you forgot the roles gtnh staff and staff on sabbatical

**Comment by lordIcocain** [comment]
> I think you forgot the roles gtnh staff and staff on sabbatical. They are basically devs too afaik
but many not dev ppl have that roles

**Comment by S4mpsa** [comment]
When does this play? If it's after you go through the end portal, it's way too premature. HV is barely earlygame.

Should really play after you make your first Eye of Harmony (or go through the stargate but that might be a little hard to achieve for most people)

**Comment by Ultraxite** [comment]
some comments:

"enjoyable" thing at the end feels a bit esoteric and goofy for such a heartfelt, serious credits sequence. it should probs be "Gregtech
New Horizons"
where the New Horizons does the remaining thing like "enjoyable" does


I think there needs to be a shoutout/description part talking about GregoriusT and GregTech 5. Without them, this pack wouldnt exist. 

Maybe also a few lines in that "now that youve beat the pack, good luck in other packs" section talking about how you've come full circle and can/should now contribute to development!

it could be cool to also have some screenshots breaking up each section of some cool loading-screen-art style shots of bases, especially endgame ones. Maybe even get some concept art or something from the devs - old videogame credits loved doing concept art/demo models etc.

**Comment by vovaksenov** [comment]
I think the text is a placeholder. It looks ai generated

**Comment by OneEyeMaker** [comment]
I think there should be *at least* 2 different screens of such kind for different events.
1) Firstly, screen that replaces vanilla one (after killing End dragon). Purpose of this screen should be "wish for good luck" for players reaches first early-game milestones.
2) Secondly, screen with content demonstrated in video. And this one should be shown only for most dedicated players who reached some endgame goal. (EoHoho?/StarGate?)

**Comment by lordIcocain** [comment]
> When does this play? If it's after you go through the end portal, it's way too premature. HV is barely earlygame.
> 
> Should really play after you make your first Eye of Harmony (or go through the stargate but that might be a little hard to achieve for most people)

My idea was play when you first time go in stargate, but then I found out that official servers using gate... I think mb just add check if player have completed stargate quest.

**Comment by lordIcocain** [comment]
> I think the text is a placeholder. It looks ai generated

yeah, i used chatgpt. I couldn't even write a sentence haha. So if someone want write better text i would be so happy.

**Comment by lordIcocain** [comment]
> I think there should be _at least_ 2 different screens of such kind for different events.
> 
> 1. Firstly, screen that replaces vanilla one (after killing End dragon). Purpose of this screen should be "wish for good luck" for players reaches first early-game milestones.
> 2. Secondly, screen with content demonstrated in video. And this one should be shown only for most dedicated players who reached some endgame goal. (EoHoho?/StarGate?)

I think first one require change original mc code. if its possible would be cool. like a ask "did you think its end? ahaha"

**Comment by vovaksenov** [comment]
I think that the stargate end screen should start if you click on the stargate trophy. It would allow people to replay the credits if they want and add a cool function to the newly added trophy.

**Comment by Ultraxite** [comment]
> .

it definitely reads like its saying a whole lot of nothing, very typical of AI writing models. I'd be happy to give it a shot at some point if nobody else does it. Dont blame you though, it can be hard if english isnt your first language

File: src/main/resources/assets/dreamcraft/RealWinGameData.txt (hunk 1 above)
Comment (lordIcocain) [reply src/main/resources/assets/dreamcraft/RealWinGameData.txt]: fixed

**Comment by Connor-Colenso** [comment]
Staff shouldn't be lumped in with devs, some staff don't dev at all. It should be a distinct category under admins.

**Comment by Dream-Master** [comment]
@lordIcocain any news here ?

**Comment by lordIcocain** [comment]
Nope

**Comment by IcaroKaue** [comment]
I made a new text for it, if anyone wants to take a look. -> https://pastebin.com/2Kh7Pb5D 

If it gets accepted I'll translate it into Portuguese.

**Comment by IcaroKaue** [comment]
> > I think there should be _at least_ 2 different screens of such kind for different events.
> > 
> > 1. Firstly, screen that replaces vanilla one (after killing End dragon). Purpose of this screen should be "wish for good luck" for players reaches first early-game milestones.
> > 2. Secondly, screen with content demonstrated in video. And this one should be shown only for most dedicated players who reached some endgame goal. (EoHoho?/StarGate?)
> 
> I think first one require change original mc code. if its possible would be cool. like a ask "did you think its end? ahaha"

@lordIcocain According to the [MC Wiki](https://minecraft.wiki/w/Credit_Sequence) you only need to change "assets/minecraft/texts/end.txt". I think a txloader file would work. (We can change the credits and quote too, maybe adding the devs and such 😀)

**Comment by Dream-Master** [comment]
@lordIcocain any news here?

**Pull request closed without merging**